packages/squidex/schema/crn/rules/entity-updated.json

```diff
@@ -36,6 +36,10 @@
       {
         "schemaId": "users",
         "condition": "event.type === 'Created' || event.type === 'Published' || event.type === 'Unpublished' || event.type === 'Updated' || event.type === 'Deleted'"
+      },
+      {
+        "schemaId": "working-groups",
+        "condition": "event.type === 'Created' || event.type === 'Published' || event.type === 'Unpublished' || event.type === 'Updated' || event.type === 'Deleted'"
```

Comment:
I guess this is in line with the rest of them but maybe we don't want to trigger the call for unpublishing and deleting?